EIPS/eip-5585.md

```diff
@@ -0,0 +1,111 @@
+---
+eip: 5585
+title: EIP-721 NFT Authorization
+description: NFT owners can authorize other users to use their NFTs for business purpose.
+author: Veega Labs (@VeegaLabsOfficial)
+discussions-to: https://ethereum-magicians.org/t/nft-authorization-erc721-extension/10661
+status: Draft
+type: Standards Track
+category: ERC
+created: 2022-08-15
+requires: 721
+---
+
+## Abstract
+
+1. By registering and confirming ownership and other commercial rights independently on the chain, it supports the independent circulation of commercial authorization. Thereby the users will get more benefits in addition to selling, and it will significantly motivate users in various ways.
+2. After separating the ownership of assets and other commercial rights, the information of full authorization chain will be saved by smart contracts, and the benefits of commercial users will be protected by smart contracts on the blockchain, and would not be affected by the changes of ownership.
+3. The cryptocurrency paid by the users will be hosted and locked in the smart contract, and the "authorized funds" will be dynamically unlocked to asset owners based on how long the assets being used. （this feature will be implemented in the future）
+
+## Motivation
+
+The development of NFT has simplified the ownership verification and transaction. However, other kinds of rights except ownership such as exhibition rights, lease rights, creation of derivatives etc. are still difficult to be circulated and used independently, which have limited the value and commercialization of digital assets.
+
+As the development of the Metaverse, digital assets are diverse on the blockchain such as digital collections, virtual real estate, and commercial copyrights etc.,  the commercial rights (exhibition rights, derivative rights, commercial authorization, etc.) based on digital assets are becoming a potential business form. 
+
+After separating the ownership of assets and other commercial rights, other problems will be raised. For example, when the ownership changes, at the same time the commercial authorization is still valid, it will result the conflict interest between the new owner and the commercial authorization user. It is extremely important thing how to protect the interests of both parties.
+
+## Specification
+
+```solidity
```

Comment:
This is not a valid solidity interface, and the indentation is off. Please fix this.